Sources/Mocker/Mock.swift

```diff
@@ -101,9 +101,13 @@ public struct Mock: Equatable {
     /// Can only be set internally as it's used by the `expectationForCompletingMock(_:)` method.
     var onCompletedExpectation: XCTestExpectation?
 
-    private init(url: URL? = nil, ignoreQuery: Bool = false, cacheStoragePolicy: URLCache.StoragePolicy = .notAllowed, dataType: DataType, statusCode: Int, data: [HTTPMethod: Data], requestError: Error? = nil, additionalHeaders: [String: String] = [:], fileExtensions: [String]? = nil) {
+    private init(url: URL? = nil, ignoreQuery: Bool = false, cacheStoragePolicy: URLCache.StoragePolicy = .notAllowed, dataType: DataType? = nil, statusCode: Int, data: [HTTPMethod: Data], requestError: Error? = nil, additionalHeaders: [String: String] = [:], fileExtensions: [String]? = nil) {
```

Comment:
Out of scope for this pull request, but I wonder putting these long line on some separate lines will also help reviewing this in the future, as it was hard to see what changed here (I assumed it was the `dataType` now defaulting to `nil`, but still!)

Reply:
Yes, I indeed thought about rewrapping these lines. But then I've seen the active branch `feature/doc-c` which applies `SwiftFormat` to the codebase. -> I do not want to introduce more git conflicts as needed.

Reply:
Well... I still do not rewrap the method signatures... but to be consistent over the initializers, I've added some wrapping to the callers.